lib/api/wpapi.js

```diff
@@ -0,0 +1,77 @@
+var $ = require('zepto-browserify').$;
+
+var name = 'WP.CORE';
+
+var baseUrl = 'https://public-api.wordpress.com/wp/v2';
```

Comment:
Or can we have an array of namespaces here and iterate over them to build the full list?  `[wp/v2, wpcom/v2]`